app/frontend/pgbus/tailwind.css

```
@@ -26,11 +43,24 @@
   & tbody td {
```

Stylelint (17.6.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)